### ISSUE TITLE
Feature request: Offline support

### ISSUE THREAD

**Issue by actuallynotjohndoe** [Teifun2/nextcloud-cookbook-flutter#48]
Hello, I'd like to suggest a feature that is currently missing in your otherwise quite nice app: offline support! I sometimes do not have internet access while I'm cooking, so it would be nice to be able to access my recipes without internet. One possible solution to this problem is to rely on the locally installed nextcloud app that can sync a folder from the cloud to the device. Luckily, this has already been implemented by another developer – in the other nextcloud cookbook client for android: https://codeberg.org/MicMun/nextcloud-cookbook
Sadly, I can't use that app because it has issues with parsing newly added recipes. So I'd like to use your app. 
Thanks in advance!

**Comment by Teifun2** [comment]
@actuallynotjohndoe Yes Offline support is definitively a feature i want to have at some time in the future. But currently i will focus on adding / editing recipes. And will then transition into caching.

Thanks for your involvement!

**Comment by Teifun2** [comment]
With the PR #77 this will be supported a little.

 If the recipe has been loaded it will stay in the cache for a while. Currently i don't know how long it will stay there but it is possible to open recipes without any internet connection.

This is however not the final solution I want to have in the end!
